### PR TITLE
Icelandic update (login screen, UI, dash), English update

### PR DESCRIPTION
**Pull request by Nammi-namm**
Icelandic: Added many news strings, mostly login screen, UI, and corrected a few things to a more unified vocabulary where there used to be inconsistencies. Changed some words based on feedback (regarding ambiguity).

English: Fixed incorrect word used for Register.MissingUID and clarified Settings.LinuxIntegration (to differentiate it from Android)

### CODE REVIEW

File: en.json [hunk 1]
```diff
@@ -1264,7 +1264,7 @@
         "Settings.ViveHandTracking.UseFingersWhenSnapped": "Use Vive Fingers When Snapped",
 
         "Settings.WindowsIntegration": "Windows Integration",
-        "Settings.LinuxIntegration": "Linux Integration",
+        "Settings.LinuxIntegration": "GNU/Linux Integration",
```
Comment (Frooxius) [inline en.json:1267]: Let's keep this colloquial please.

**Review by Frooxius — CHANGES_REQUESTED** [review]
I'd strongly suggest de-coupling changes for multiple languages, so they do not hold it up.

**Comment by Nammi-namm** [comment]
Changed back as requested.

>I'd strongly suggest de-coupling changes for multiple languages, so they do not hold it up.

I'll keep this in mind thanks.

**Comment by Frooxius** [comment]
Thanks, merging in!

**Pull request merged**